test/unit/workflows/test_modules.py

```diff
@@ -1,5 +1,5 @@
 import json
-from collections import namedtuple
+from typing import Any, Dict, List, NamedTuple, Optional, Tuple, Union
```

Comment:
More a question than a suggestion... do we have rules defined in the style guide about when we should use the multiline import instead of inline?

```suggestion
from typing import (
    Any,
    Dict,
    List,
    NamedTuple,
    Optional,
    Tuple,
    Union,
)
```

Reply:
I'm not sure we really have a policy - I think black would enforce one https://github.com/galaxyproject/galaxy/pull/11162. My rule of thumb is switching to the per-line version on import 3 or when I'm confident the list of imports will change a lot over time. This piece of code meets both of those criteria and I have made the requested changes.